src/screens/videos/videos.css

```diff
@@ -4,3 +4,14 @@
 		url("../../../assets/images/videos-you-might-like/penguins.png") bottom right/auto no-repeat,
 		url("../../../assets/images/videos-you-might-like/sky.jpg") center/cover no-repeat;
 }
+
+/* For Webkit-based browsers (Chrome, Safari and Opera) */
+.scrollbar-hide::-webkit-scrollbar {
```

Comment:
I added this to enable horizontal touch scrolling without seeing a scrollbar. There might be a better way? This could me made global if we need it elsewhere.

Reply:
There's a Tailwind plugin for styling scrollbars. Might be worth looking into that: https://www.npmjs.com/package/tailwind-scrollbar.

Reply:
Good idea & done!